sdk/storage/AzureStorageBlob/Source/StorageStreamDownloader.swift

```diff
@@ -87,14 +82,6 @@ internal class ChunkDownloader {
     ///   - requestId: Unique request ID (GUID) for the operation.
     ///   - completion: A completion handler that forwards the downloaded data.
     public func download(requestId: String? = nil, completion: @escaping HTTPResultHandler<Data>) {
-        // Construct URL
-        let urlTemplate = "/{container}/{blob}"
```

Comment:
I'm okay with this change, but this pattern was lifted from the Python SDK and is geared toward being something that could be procedurally code-generated in the future.

Reply:
Resolved, will push after we chat.